python/examples/01_tensorflow-basic.ipynb

```diff
@@ -0,0 +1,290 @@
+    "                               input_data_type = 'NUMPY',\n",
+    "                               output_data_type = 'NUMPY')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `ServingConfig` has to be passed to `InferenceConfiguration`, in addition to the pipeline steps as a Python list. In this case, there is a single pipeline step: `tf_pipeline_step`. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "server = Server(config = InferenceConfiguration(serving_config = serving_config,\n",
+    "                                                pipeline_steps = [tf_pipeline_step]))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "By default, `Server()` looks for the Konduit Serving JAR `konduit.jar` in the directory the script is run in. To change this default, use the `jar_path` argument."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The configuration is stored as a dictionary. Note that the configuration can be converted to a dictionary using the `as_dict()` method: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "server.config.as_dict()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configuring the client \n",
+    "\n",
+    "To configure the client, create a Client object with the following arguments: \n",
+    "- `input_names`: names of the input data\n",
+    "- `output_names`: names of the output data\n",
+    "- `input_type`: data type passed to the server for inference\n",
+    "- `endpoint_output_type`: data type returned by the server endpoint \n",
+    "- `return_output_type`: data type to be returned to the client\n",
+    "\n",
+    "\n",
+    "<div class=\"alert alert-warning\">\n",
+    "    ⚠ Future versions of the Python SDK may remove the <code>input_names</code> and <code>output_names</code> arguments in <code>Client()</code>, since these are already specified in <code>ModelPipelineStep()</code>. \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = Client(input_names = input_names,\n",
+    "                output_names = output_names,\n",
+    "                input_type = 'NUMPY',\n",
+    "                endpoint_output_type = 'NUMPY',\n",
+    "                return_output_type = \"NUMPY\",\n",
+    "                url = 'http://localhost:' + str(port))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Running the server "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We load some sample data from NumPy files. Note that these are NumPy arrays, each with shape (4, 128): "
```

Comment:
"We will load" instead of "We load"

Reply:
Reworded to be "Load some sample data ..." to be consistent with the rest of the document.